Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -107,14 +120,14 @@ class KeyboardAvoidingView extends React.Component<Props, State> {
     }
   };
 
-  _updateBottomIfNecessary = () => {
+  _updateBottomIfNecessary = async () => {
```

Comment:
Call sites for function `_updateBottomIfNecessary` is not using await, as a side effect the function call `this.props.onLayout(event)` now won't wait for `_updateBottomIfNecessary` to finish.

As per the doc, `This event is fired immediately once the layout has been calculated`

Reply:
Ohh good point, I added an `await` to the `_updateBottomIfNecessary` call to ensure everything is properly calculated before the `this.props.onLayout` function is called.